openeo/rest/datacube.py

```diff
@@ -2732,9 +2739,10 @@ def sar_backscatter(
         .. versionadded:: 0.4.9
         .. versionchanged:: 0.4.10 replace `orthorectify` and `rtc` arguments with `coefficient`.
         """
-        coefficient_options = [
-            "beta0", "sigma0-ellipsoid", "sigma0-terrain", "gamma0-ellipsoid", "gamma0-terrain", None
-        ]
+        coefficient_options = [None]
+        if self.connection:
+            schema = self.connection.get_schema_from_process_parameter("sar_backscatter", "coefficient")
+            coefficient_options += search_list_for_dict_key(schema, "enum")
```

Comment:
I think it's risky to put this new logic in the main code path here. I would put a try-except around it because this could still go wrong in various places (both in metadata provided by backend and client side assumptions). When something goes wrong we should show a warning about this, but fallback on the original hard-coded coefficient list.